pyproject.toml

```diff
@@ -1,10 +1,62 @@
 [build-system]
 requires = [
     "pkgconfig",
-    "setuptools >= 46.4.0",
+    "setuptools >= 67.0",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name        = "aiohttp"
+license     = {text = "Apache-2.0 AND MIT"}
```

Comment:
This is incorrect. SPDX must not be added as license text: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license

Reply:
Okay. Marking this chain as resolved.